src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -351,7 +350,7 @@ public String getQuestionAdditionalInfoHtml(int questionNumber,
     public String getQuestionResultStatisticsHtml(
             List<FeedbackResponseAttributes> responses,
             FeedbackQuestionAttributes question,
-            PageData pageData,
+            String studentEmail,
```

Comment:
It's because of abstract method. The `studentEmail` is used for contribution and numscale questions only but the abstract method makes it necessary for every single `getQuestionResultStatisticsHtml` method to include that parameter.